Various diaries fail to Initalize when using `-ea`

varrock, karamja, ardougne medium all puke out this, I did not test every diary

https://github.com/Zoinkwiz/quest-helper/blob/39f3e223dce5f9803e4a5918efdfbf38cb061e5e/src/main/java/com/questhelper/panel/QuestOverviewPanel.java#L304 probably to blame 
```
2023-11-20 17:36:18 EST [AWT-EventQueue-0] ERROR net.runelite.client.RuneLite - Uncaught exception:
java.lang.AssertionError: must be called on client thread
        at client.getVarbitValue(client.java:23649)
        at client.getVarbitValue(client.java:16143)
        at com.questhelper.requirements.var.VarbitRequirement.check(VarbitRequirement.java:153)
        at com.questhelper.requirements.conditional.Conditions.lambda$check$0(Conditions.java:126)
        at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:178)
        at java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1625)
        at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509)
        at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
        at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:921)
        at java.base/java.util.stream.ReduceOps$5.evaluateSequential(ReduceOps.java:258)
        at java.base/java.util.stream.ReduceOps$5.evaluateSequential(ReduceOps.java:248)
        at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
        at java.base/java.util.stream.ReferencePipeline.count(ReferencePipeline.java:709)
        at com.questhelper.requirements.conditional.Conditions.check(Conditions.java:127)
        at com.questhelper.requirements.conditional.Conditions.lambda$check$0(Conditions.java:126)
        at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:178)
        at java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1625)
        at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509)
        at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
        at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:921)
        at java.base/java.util.stream.ReduceOps$5.evaluateSequential(ReduceOps.java:258)
        at java.base/java.util.stream.ReduceOps$5.evaluateSequential(ReduceOps.java:248)
        at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
        at java.base/java.util.stream.ReferencePipeline.count(ReferencePipeline.java:709)
        at com.questhelper.requirements.conditional.Conditions.check(Conditions.java:127)
        at com.questhelper.panel.QuestOverviewPanel.addQuest(QuestOverviewPanel.java:304)
        at com.questhelper.panel.QuestHelperPanel.addQuest(QuestHelperPanel.java:470)
        at com.questhelper.managers.QuestManager.lambda$initializeNewQuest$5(QuestManager.java:304)
        at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:318)
        at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:773)
        at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:720)
        at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:714)
        at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
        at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
        at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:742)
        at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
        at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
        at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
        at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
        at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
        at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
```

--- a/src/main/java/com/questhelper/managers/QuestManager.java
+++ b/src/main/java/com/questhelper/managers/QuestManager.java
@@ -299,10 +299,8 @@ public class QuestManager
 				return;
 			}
 			questBankManager.startUpQuest();
-			SwingUtilities.invokeLater(() -> {
-				panel.removeQuest();
-				panel.addQuest(questHelper, true);
-			});
+			panel.removeQuest();
+			panel.addQuest(questHelper, true);
 		}
 		else
 		{

